browser/views.py

```diff
@@ -687,3 +686,47 @@ def delete_investigator_record(request, person_id, research_id):
         Investigator.objects.filter(id=research_id).delete()
 
     return HttpResponseRedirect(reverse('person', args=(person_id,)))
+
+
+@staff_member_required
+def position(request, person_id, position_id):
+    person = get_object_or_404(Person, pk=person_id)
+    template = "browser/position.html"
+    form = PositionForm()
+    context = {
+        'form': form,
+        'person': person,
+    }
+
+    if position_id is not None:
+        position = get_object_or_404(Position, pk=position_id)
+        form = PositionForm(initial={'subject': position.subject, 'role': position.role,
+                                     'year': position.year, 'start_date': position.start_date,
+                                     'end_date': position.end_date})
+        context.update({
+            'position_id': position_id,
+            'form': form
+        })
+
+    if request.method == 'POST':
+        form = PositionForm(request.POST, instance=person)
```

Comment:
if there is a position id, then ```form``` is already set, isn't it?

Reply:
Yes, the form is already set if there is a position_id .  So, I have removed ```instance=person```